### PR TITLE
2027 column settings

### PR DESCRIPTION
**Pull request by aviveiros11**
#### Any background context you want to provide?
Giving more instruction to the user.  Making an official PR for this because I am unsure of where to go from here!
#### What's this PR do?
Modifies the instruction layout on the Column settings page and adds a tabular function to collapse it
#### How should this be manually tested?
on the Column Settings page review the text and collapse the text
#### What are the relevant tickets?
#2027

### CODE REVIEW

File: seed/static/seed/js/controllers/column_settings_controller.js [hunk 1]
```diff
@@ -66,6 +67,14 @@ angular.module('BE.seed.controller.column_settings', [])
         {id: 'geometry', label: $translate.instant('Geometry')}
       ];
 
+      $scope.changeText = function(btnText) {
+        console.log(btnText);
```
Comment (adrian-lara) [inline seed/static/seed/js/controllers/column_settings_controller.js]: Haven't reviewed fully, but noticed this console.log

Comment (aviveiros11) [reply seed/static/seed/js/controllers/column_settings_controller.js]: whoops, thank you!

**Comment by aviveiros11** [comment]
Hey, @adrian-lara yeah the initial state was messed up with the collapse :) thanks!

**Comment by RDmitchell** [comment]
@aviveiros11 / @adrian-lara / @axelstudios 

The formatting of this text is a little hard to read -- the bullets don't really line up with their subject to the left in a good way. 

Is it possible to add a line between each topic or something like that that makes it easier to read?

**Comment by axelstudios** [comment]
@aviveiros11 Here's a diff of my working dir that might make the formatting a bit cleaner:
```

File: seed/static/seed/partials/column_settings.html [hunk 2]
```diff
@@ -27,9 +27,9 @@ <h2><i class="fa fa-sliders"></i> <span translate>Column Settings</span></h2>
     </div>
     <div class="section_content_container">
         <div class="section_content with_padding" style="padding: 15px;">
-            <button type="button" class="btn btn-primary" data-toggle="collapse" data-target="#instruction" ng-click="changeText(btnText)">{$ btnText $}</button>
+            <button type="button" class="btn btn-primary" data-toggle="collapse" data-target="#instruction" ng-click=changeText(btnText)>{$ btnText $}</button>
         </div>
-        <div class="section_content with_padding" style="margin-bottom:15px;"  id="instruction" class="collapse">
+        <div class="section_content with_padding collapse in" style="margin-bottom:15px;" id="instruction">
```
[the remark this answers is not in the document]
Comment (aviveiros11) [reply seed/static/seed/partials/column_settings.html]: @adrian-lara I read that the `collapse in` class shows the div by default.  Do you want the instructions hidden on page load?  

> By default, the collapsible content is hidden. However, you can add the .in class to show the content by default:

**Comment by aviveiros11** [comment]
@axelstudios I threw in what was in your working directory however I still don't think it solves @RDmitchell 's concern of readability.  I will look into adding a spacer for each row.

**Comment by RDmitchell** [comment]
@aviveiros11 -- actually, @axelstudios change makes it much better, because the first bullet always lines up with the first line of the subject. 

So that might be good enough for now.

Robin

**Comment by axelstudios** [comment]
You can also change the `padding-bottom: 5px;` line to increase the padding between each section

**Comment by aviveiros11** [comment]
Great! should be all set now :)

File: seed/static/seed/partials/column_settings.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (adrian-lara) [reply seed/static/seed/partials/column_settings.html]: Ah - didn't know that! 👍

**Pull request merged**